### ISSUE TITLE
rRootage gets placed on different workspace on start

### ISSUE THREAD

**Issue by damianatorrpm** [WayfireWM/wayfire#961]
Application: com.abagames.rRootage

This games when started goes into fullscreen for about 100 ms than disappears.
One would think that it crashed but it moved (itself?) to another workspace.

It doesn't show this behaviour on gnome. Will test without place plugin and write here.

**Comment by damianatorrpm** [comment]
Without the place plugin it also moves to another workspace (never appears on the one you're currently at), but the workspace is different (the last instead of next of current)

**Comment by chaoskagami** [comment]
I just installed rRootage (`emerge rrootage`) and I can't reproduce this one on my system at least - running git master wayfire. It starts on the current workspace perfectly fine, whether I make it run windowed (`rrootage -window`) or not. Plugins list for me is `autostart wm-actions command expo fast-switcher grid move oswitch place resize vswitch switcher window-rules idle force-fullscreen winzoom workspace-names`.

This game also uses SDL 1.2 instead of SDL2, so it doesn't even support Wayland natively; it's running under Xwayland.

**Comment by ammen99** [comment]
@chaoskagami You probably need to do this: open rRootage on one monitor, close it, focus the other monitor, open rRootage there. I expect the bug will happen.

**Comment by chaoskagami** [comment]
Ah. So it's related to having mutliple monitors, then. That explains why I can't repro it - I only have one on this machine.

**Comment by ammen99** [comment]
> I also may have conflated "viewports" and "workspaces" by accident, sorry.

Viewports and workspaces are the same thing in wayfire :)

**Comment by killown** [comment]
This game is untestable, too outdated to run on most distros. This issue is invalid. Also there is a huge possibility the issue is already fixed.

**Comment by soreau** [comment]
Yes, I agree that this is an old issue and probably isn't relevant anymore. Closing.
